### ISSUE TITLE
Cannot rename jobs

### ISSUE THREAD

**Issue by majblackburn** [crmgustaf/BulkDeleteManager#6]
It appears to intend to allow you to rename jobs (wanted to add "ZZ" to the end of inactive ones to get them out of the way), but when clicking "save" I get the message "You have not selected create new Bulk Delete job."

**Comment by majblackburn** [comment]
That said, allowing us to delete canceled jobs would be even better for my particular concern, but that's more a feature request

**Comment by crmgustaf** [comment]
The problem is that the platform doesn't allow changing names of recurring bulk delete jobs. I also havn't found a way of being able to delete a cancelled recurring job. So, I think the most resonable short term fix would be to just disable the "renaming" feature.

I have tried to get hold of someone at MSFT that owns the bulk delete functionality but I havn't gotten anywhere, as I have many things I would like to have enhanced with it.

**Comment by crmgustaf** [comment]
> It appears to intend to allow you to rename jobs (wanted to add "ZZ" to the end of inactive ones to get them out of the way), but when clicking "save" I get the message "You have not selected create new Bulk Delete job."

Yes, this is a bug. The API doesn't allow changing so I should block that so it doesn't look like you can. The only thing that the API allows is creating new recurring jobs and cancelling them.

There is also an undocumented API for changing the recurrance and I am considering adding support to that. It doesn't allow changing the name though.
